src/Chorus/UI/Misc/ServerSettingsControl.cs

```diff
@@ -20,11 +20,7 @@ public ServerSettingsControl()
 		{
 			InitializeComponent();
 
-			_serverLabel.Visible = ServerSettingsModel.IsQaServer || ServerSettingsModel.IsPrivateServer;
-			if (_serverLabel.Visible)
-			{
-				_serverLabel.Text = string.Format(_serverLabel.Text, $"{(ServerSettingsModel.IsPrivateServer ? "private" : null)}{ServerSettingsModel.LanguageForgeServer}");
-			}
+			_serverLabel.Text = string.Format(_serverLabel.Text, $"{(ServerSettingsModel.IsPrivateServer ? "private" : "public")}{ServerSettingsModel.LanguageForgeServer}");
```

Comment:
I'm surprised we don't need/want a space after `private`/`public`, but that's nothing new...

Reply:
The formatted text is "to public.languageforge.org" or "to private-qa.languageforge.org" or similar.